Assets/Scripts/QuestEditor/EditorSelectionList.cs

```diff
@@ -133,7 +134,7 @@ public void SelectItem(UnityEngine.Events.UnityAction call)
             if (filtered.Count > i)
             {
                 string key = filtered[i].name;
-                tb = new TextButton(new Vector2(21, offset), new Vector2(20, 1), key, delegate { SelectComponent(key); }, filtered[i].color);
+                tb = new TextButton(new Vector2(21, offset), new Vector2(20, 1), new StringKey(key,false), delegate { SelectComponent(key); }, filtered[i].color);
```

Comment:
I don't think this should be translated, it is mostly used for references to quest items.